src/components/table/table.tsx

```diff
@@ -79,7 +79,11 @@ export const Table: FC<TableComponentProps> = ({
   const getSortIcon = (columnKey: string) => {
     if (!sortableColumns.includes(columnKey)) return;
     if (sortingColumn?.key === columnKey) {
-      return sortingDirection === SortDirection.ASC ? <ArrowUpIcon /> : <ArrowDownIcon />;
+      return sortingDirection.toLowerCase() === SortDirection.ASC ? (
```

Comment:
Can we remove `SortDirection`?
Maybe utilize the approach you used in service UI with constants for sorting directions and the types based on them?